### ISSUE TITLE
Issue while setting up `selectedBackgroundColor` and `fontSize` properties to the `SegmentedBar`

### ISSUE THREAD

**Issue by tsonevn** [NativeScript/nativescript-angular#649]
Setting up `selectedBackgroundColor`, `fontSize` in CSS or inline in the HTML will not change the `SegmentedBar` style. I am attaching sample code, where the problem could be reproduced. This problem appears only on `Android`.

app.component.html
```
<StackLayout class="p-20">
   <SegmentedBar [items]="segmentedbaritems">
   </SegmentedBar>
</StackLayout>
```

app.component.ts
```
import { Component } from "@angular/core";

@Component({
    selector: "my-app",
    templateUrl: "app.component.html",
})
export class AppComponent {
    public counter: number = 16;
    public segmentedbaritems = [{title:"title1"}, {title:"title2"}, {title:"title3"}, {title:"title4"}, {title:"title5"}];

    public get message(): string {
        if (this.counter > 0) {
            return this.counter + " taps left";
        } else {
            return "Hoorraaay! \nYou are ready to start building!";
        }
    }
    
    public onTap() {
        this.counter--;
    }
}
```

app.css
```
SegmentedBar{
    selected-background-color:yellow;
    font-size: 10;
    background-color: red
}

```

package.json

```
{
  "description": "NativeScript Application",
  "license": "SEE LICENSE IN <your-license-filename>",
  "readme": "NativeScript Application",
  "repository": "<fill-your-repository-here>",
  "nativescript": {
    "id": "org.nativescript.issue3244ng",
    "tns-ios": {
      "version": "2.4.0"
    },
    "tns-android": {
      "version": "2.4.1"
    }
  },
  "dependencies": {
    "@angular/common": "2.3.1",
    "@angular/compiler": "2.3.1",
    "@angular/core": "2.3.1",
    "@angular/forms": "2.3.1",
    "@angular/http": "2.3.1",
    "@angular/platform-browser": "2.3.1",
    "@angular/platform-browser-dynamic": "2.3.1",
    "@angular/router": "3.3.1",
    "nativescript-angular": "1.3.0",
    "nativescript-theme-core": "^0.2.1",
    "reflect-metadata": "~0.1.8",
    "rxjs": "5.0.0-rc.4",
    "tns-core-modules": "2.4.4"
  },
  "devDependencies": {
    "babel-traverse": "6.22.1",
    "babel-types": "6.22.0",
    "babylon": "6.15.0",
    "lazy": "1.0.11",
    "nativescript-dev-android-snapshot": "^0.*.*",
    "nativescript-dev-typescript": "^0.3.2",
    "typescript": "~2.0.10",
    "zone.js": "~0.7.2"
  }
}
```

**Comment by tsonevn** [comment]
The problem is fixed in NativeScipt 3.0

**Comment by SvetoslavTsenov** [comment]
The issue is already resolved with tns-core-modules@3.1.0
